Document `module_version` of manifest package version (#260)
discord-userdoccers/discord-userdoccers: pages/topics/client-distribution.mdx

###### Manifest Package Version Structure

| Name            | Type                             | Description                                               |
| --------------- | -------------------------------- | --------------------------------------------------------- |

> | host_version    | array[integer, integer, integer] | The host version that the package requires / is for       |

Reviewer: ```suggestion
| host_version    | array[integer, integer, integer] | The host version that the package targets                 |
```
